addon/components/polaris-date-picker/month.js

```diff
@@ -9,152 +14,153 @@ import {
   getWeekdaysOrdered,
 } from '../../utils/dates';
 
-export default Component.extend({
-  attributeBindings: ['role'],
-
-  classNames: ['Polaris-DatePicker__Month'],
-
-  layout,
-
+@classNames('Polaris-DatePicker__Month')
+@templateLayout(layout)
+export default class Month extends Component {
   /**
    * @property focusedDate
    * @public
    * @type {Date}
    * @default null
    */
-  focusedDate: null,
+  focusedDate = null;
 
   /**
    * @property selected
    * @public
    * @type {Object}
    * @default null
    */
-  selected: null,
+  selected = null;
 
   /**
    * @property hoverDate
    * @public
    * @type {Date}
    * @default null
    */
-  hoverDate: null,
+  hoverDate = null;
 
   /**
    * @property month
    * @public
    * @type {Number}
    * @default null
    */
-  month: null,
+  month = null;
 
   /**
    * @property year
    * @public
    * @type {Number}
    * @default null
    */
-  year: null,
+  year = null;
 
   /**
    * @property disableDatesBefore
    * @public
    * @type {Date}
    * @default null
    */
-  disableDatesBefore: null,
+  disableDatesBefore = null;
 
   /**
    * @property disableDatesAfter
    * @public
    * @type {Date}
    * @default null
    */
-  disableDatesAfter: null,
+  disableDatesAfter = null;
 
   /**
    * @property allowRange
    * @public
    * @type {Boolean}
    * @default false
    */
-  allowRange: false,
+  allowRange = false;
 
   /**
    * @property weekStartsOn
    * @public
    * @type {String}
    * @default null
    */
-  weekStartsOn: null,
+  weekStartsOn = null;
 
   /**
    * @property onChange
    * @public
    * @type {Function}
    * @default noop
    */
-  onChange(/* dateRange */) {},
+  onChange /* dateRange */() {}
 
   /**
    * @property onHover
    * @public
    * @type {Function}
    * @default noop
    */
-  onHover(/* hoverEnd */) {},
+  onHover /* hoverEnd */() {}
 
   /**
    * @property onFocus
    * @public
    * @type {Function}
    * @default noop
    */
-  onFocus(/* date */) {},
+  onFocus /* date */() {}
 
   /**
    * @property monthName
    * @public
    * @type {Function}
    * @default noop
    */
-  monthName(/* month */) {},
+  monthName /* month */() {}
 
   /**
    * @property weekdayName
    * @public
    * @type {Function}
    * @default noop
    */
-  weekdayName(/* weekday */) {},
+  weekdayName /* weekday */() {}
```

Comment:
lol, arg comment got messed up here 🤷‍♂

Reply:
Use `/** weekday */` for commenting args. Using a single asterisk can throw off many tools

Reply:
TIL 😄